### PR TITLE
Update screen mirroring in Infotainment.vspec

### PR DESCRIPTION
**Pull request by spshin3**
Add screen mirroring in infortainment.

Mirror phone screen to car screen.

### CODE REVIEW

File: spec/Cabin/Infotainment.vspec [hunk 1]
```diff
@@ -266,4 +266,27 @@ SmartphoneProjection.SupportedMode:
...
+  datatype: string
+  type: actuator
+  allowed: ['USB', 'BLUETOOTH', 'WIFI']
+  description: Connectivity source selected for mirroring.
+
+SmartphoneScreenMirroring.Resolution:
+  datatype: string
+  type: attribute
+  allowed: [ '720p', '1080p', '4K' ]
+description: Display resolution for the mirrored content.
```
Comment (erikbosch) [inline spec/Cabin/Infotainment.vspec:290]: Wrong indentation, causes build error

File: spec/Cabin/Infotainment.vspec [hunk 2]
```diff
@@ -266,4 +266,27 @@ SmartphoneProjection.SupportedMode:
...
+SmartphoneScreenMirroring.Source:
+  datatype: string
+  type: actuator
+  allowed: ['USB', 'BLUETOOTH', 'WIFI']
+  description: Connectivity source selected for mirroring.
+
+SmartphoneScreenMirroring.Resolution:
+  datatype: string
+  type: attribute
+  allowed: [ '720p', '1080p', '4K' ]
```
Comment (erikbosch) [inline spec/Cabin/Infotainment.vspec:289]: We have a recommendation at https://covesa.github.io/vehicle_signal_specification/rule_set/data_entry/allowed/#recommendation-on-string-values to use "enum-like" identifiers, i.e. something like `[A-Z][A-Z0-9_]*`.

If we want to follow that recommendation (which is a recommendation so we can ignore it if we have good reasons) we could consider having values like `['RES_720P', 'RES_1080P', 'RES_4K']` or something similar

**Comment by erikbosch** [comment]
There i a build error and the commit is not signed off. Can be solved by amending you commit with a signoff with `git commit --amend -s` and then do a force push.

**Comment by erikbosch** [comment]
Meeting notes: Please review

**Comment by spshin3** [comment]
To resolve a DCO issue, this commit will close and I will re-commit this PR.

**Pull request closed without merging**